lib/Backend/Commands.hs

```diff
@@ -360,15 +326,8 @@ buildCopyOperations oldPath newPath force = do
       CopyOperation old (new & dateModified .~ nowUtc)
 
     -- | Performs a check on all `CopyOperation`s and throws an error if any of checks fail.
```

Comment:
Since you changed what this function does, this haddock is now outdated.

Reply:
Done